docs/pom.xml

```diff
@@ -52,6 +44,83 @@
 			<id>docs</id>
 			<build>
 				<plugins>
+					<plugin>
```

Comment:
Why is this whole section now needed?

Reply:
There are 7 dependencies that's missing in total. 5 of them install by this `maven-install-plugin` from zip file. Another 2 are pre-installed into `local-maven-repo` because they are needed in `asciidoctor-maven-plugin` and `maven-install-plugin` installations are not available to it.